### PR TITLE
feat: add some RNA QC

### PR DESCRIPTION
**Pull request by marrip**

### CODE REVIEW

**Comment by marrip** [comment]
I still have to add test files and adapt schemas so on but mabe you can have a look at multiqc and check if this could be a solution for the RNA/DNA problem we had.

**Comment by jonca79** [comment]
It works if we only have RNA in the run but if it is both won't it fail?

**Comment by marrip** [comment]
Looks really good! I like it! 🎉

**Comment by Smeds** [comment]
New version of multiqc have been merged into develop

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Changelog
```
Comment (Smeds) [inline CHANGELOG.md:1]: Is this files supposed to be added here? It's already in master.

File: workflow/rules/common.smk [hunk 2]
```diff
@@ -56,4 +56,22 @@ def get_flowcell(units, wildcards):
 
 def compile_output_list(wildcards):
     output_files = ["qc/multiqc/multiqc.html"]
```
Comment (Smeds) [inline workflow/rules/common.smk:58]: Needs to be updated to match the latest version of QC!

File: workflow/rules/multiqc.smk [hunk 3]
```diff
@@ -11,6 +11,7 @@ rule multiqc:
             for file in config["multiqc"]["qc_files"]
             for sample in get_samples(samples)
             for u in units.loc[sample].dropna().itertuples()
+            if u.type in config.get("multiqc", "").get("types", ["T", "N"])
```
Comment (Smeds) [inline workflow/rules/multiqc.smk:14]: Not needed with the latest version of QC

File: CHANGELOG.md (hunk 1 above)
Comment (marrip) [reply CHANGELOG.md:1]: I did a rebase with master at some point and now I have in develop as well 😅  Guess it's not causing any harm, is it? 🥺

Comment (marrip) [reply CHANGELOG.md:1]: I did a rebase with master at some point and now I have in develop as well 😅  Guess it's not causing any harm, is it? 🥺

**Review by Smeds — APPROVED** [review]

**Pull request merged**